staging/config/unclassified.yaml

```diff
@@ -1,5 +1,14 @@
 unclassified:
   gitHubPluginConfig:
   jiraGlobalConfiguration:
+    sites:
+        - credentialsId: "JIRA_PLUGIN"
```

Comment:
Please always use the same indentation (2 spaces), as in https://github.com/nuxeo/jx-platform-jenkins/blob/master/config/unclassified.yaml#L8.

Reply:
ok will try with undefined  credential